### PR TITLE
Update tutorial-asp-net-core-with-react.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes

### CODE REVIEW

File: docs/javascript/tutorial-asp-net-core-with-react.md [hunk 1]
```diff
@@ -63,9 +63,11 @@ You can use the method described in this article to create ASP.NET Core Single P
 
 ## Set the project properties
 
-1. In Solution Explorer, right-click the ASP.NET Core project and choose **Properties**.
+1. In Solution Explorer, right-click the ***ReactWithASP.Server*** project and choose **Properties**.
 
-   :::image type="content" source="media/vs-2022/asp-net-core-project-properties.png" alt-text="Screenshot showing Open project properties"::: 
+   :::image type="content" source="media/vs-2022/react-choose-template2.png" alt-text="Screenshot showing choosing a template":::
```
Comment (Rick-Anderson) [inline docs/javascript/tutorial-asp-net-core-with-react.md:68]:
```suggestion
   :::image type="content" source="media/vs-2022/react-choose-template2.png" alt-text="Screenshot showing choosing a template again":::
```

**Comment by Court72** [comment]
Hi @rick-anderson - This pull request was opened in the public repo. Content developers should work in the private repo, per the Microsoft Docs contributor guide. Would you make this content update in the private repo instead? Thank you! We'll close this PR.

**Comment by Rick-Anderson** [comment]
That's OK. You can finish it.

**Comment by Rick-Anderson** [comment]
@Court72 I can't believe you're going to flush all this work. It's a vast improvement over the current tutorial that doesn't work. Unfortunately I don't have time to move this to the PR repo so all this effort is wasted and you have a tutorial that's wrong, hard to follow, and ambiguous (open the project, what project, there's two projects) 

cc @Mikejo5000

**Comment by Mikejo5000** [comment]
@Rick-Anderson I can plan to port the updates. I will need to finish up various work items first though, so it might be a little while before I can get back to it.
